src/next-collection-view.js

```diff
@@ -577,6 +577,17 @@ const CollectionView = Backbone.View.extend({
     this.Dom.appendContents(this.el, els, {_$el: this.$el});
   },
 
+  swapChildViews(view1, view2) {
+    if (!this.children.hasView(view1) || !this.children.hasView(view1)) {
```

Comment:
This is nice for drag and drop view swapping or something similar.
Does this need something like a DOM parent check for both views to make sure they are being swapped from one dom parent to another? 
Or can this support something like, view1 has no parent but will swap with view2 which has a parent. Not sure what will happen with view2. I guess the person using the function will have to figure how to handle that.